pkg/services/ngalert/api/api_ruler.go

```diff
@@ -265,6 +269,18 @@ func (srv RulerSrv) RoutePostNameRulesConfig(c *contextmodel.ReqContext, ruleGro
 	return srv.updateAlertRulesInGroup(c, groupKey, rules)
 }
 
+func (srv RulerSrv) checkGroupLimits(group apimodels.PostableRuleGroupConfig) error {
```

Comment:
We'll need a better abstraction than this around the limits, but this is sufficient for our warning for now. We'll iterate and improve this over time.